nginx.conf

```diff
@@ -0,0 +1,40 @@
+server {
+    listen 8080;
+    server_name localhost;
+    absolute_redirect off;
+
+    location / {
+        gzip on;
+        gzip_http_version 1.0;
+        gzip_comp_level 5;
+        gzip_proxied any;
+        gzip_min_length 500;
+        gzip_vary on;
+        gzip_disable "MSIE [1-6]\.";
+        gzip_types application/atom+xml application/javascript application/json application/rss+xml
+          application/vnd.ms-fontobject application/x-font-ttf application/x-web-app-manifest+json application/xhtml+xml
+          application/xml font/opentype image/svg+xml image/x-icon text/css text/plain text/x-component;
+        
+        root /usr/share/nginx/html;
+        index index.html;
+        try_files $uri $uri/ /index.html =404;
+        add_header Cache-Control "no-store, no-cache, must-revalidate";
+    }
+
+    location /health {
+        return 200 'alive';
+        add_header Content-Type text/plain;
+    }
+
+    location /check_link/ {
```

Comment:
To powinno być pod prefixem /api/. Przy takiej konfiguracji przy dodawaniu nowego endpointu musisz modyfikować usługę API i jeszcze konfigurację nginx.